api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -165,13 +165,12 @@
                 .reviewSize(participantCohortStatuses.size())
                 .reviewedCount(0L)
                 .reviewStatus(ReviewStatus.CREATED);
-        List<ParticipantCohortStatus> paginatedPCS = participantCohortStatuses
-                .stream()
-                .limit(PAGE_SIZE)
-                .collect(Collectors.toList());
 
         cohortReviewService.saveFullCohortReview(cohortReview, participantCohortStatuses);
```

Comment:
nit: Perhaps worth a comment here or on `saveFullCohortReview()` that this is mutating its parameters; not obvious that the order would matter from just reading this code here.